src/util/PolarUtils.ts

```diff
@@ -39,7 +50,7 @@ export const getMaxRadius = (
  * @return {Object} Configuration
  */
 export const formatAxisMap = (
-  props: any,
+  props: FormatAxisMapProps,
```

Comment:
The props are the props of axis itself, right? 

```
    Object.keys(axisObj).forEach(key => {
      axisObj[key] = formatAxisMap(props, axisObj[key], offset, key.replace('Map', ''), chartName);
    });
```suggestion
  props: FormatAxisMapProps,
```
```

Reply:
On closer look - these appear to be `CategoricalChartProps`.